.github/workflows/loderunner-api.yaml

```diff
@@ -75,13 +75,8 @@ jobs:
       if: ${{ inputs.run_and_test }}
       run: |
         cd src/LodeRunner.API.Test
-        mkdir -p temp/secrets
-        echo -n ${{ secrets.cosmos_rw_key }} > temp/secrets/CosmosKey
-        echo -n ${{ secrets.cosmos_url }} > temp/secrets/CosmosUrl
-        echo -n 'LodeRunner' > temp/secrets/CosmosCollection
-        echo -n 'LodeRunnerTestDB' > temp/secrets/CosmosDatabase
-        dotnet test --filter "Category=Integration" -o temp
-
+        dotnet test --filter "Category=Integration"
```

Comment:
I know we talked about consolidation but don't we also need way to have secrets from /tmp/secrets before doing so?